src/Package/Impl/ProjectSystem/RProjectLoadHooks.cs

```diff
@@ -72,7 +72,8 @@ internal sealed class RProjectLoadHooks {
             , IRToolsSettings toolsSettings
             , IThreadHandling threadHandling
             , ISurveyNewsService surveyNews
-            , [Import(AllowDefault = true)] IProjectItemDependencyProvider dependencyProvider) {
+            , [Import(AllowDefault = true)] IProjectItemDependencyProvider dependencyProvider
+            , ICoreShell coreShell) {
```

Comment:
Let's just don't do anything with that `GetGlobalService` usage for now.

---
In reply to: [80995237](https://github.com/Microsoft/RTVS/pull/2408#discussion_r80995237) [](ancestors = 80995237)